src/serialport.cpp

```diff
@@ -172,74 +171,73 @@ void EIO_AfterOpen(uv_work_t* req) {
 }
 
 NAN_METHOD(Update) {
-  NanScope();
```

Comment:
No, anything declared with `NAN_METHOD()` (or other `NAN_*()`) doesn't need it, that's the point of the text you copy and pasted above.